modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -35,12 +32,10 @@ def self.parse_file(infile)
           parts[partname] = body
           break unless moreparts
         end
-      ensure
-        input.close
```

Comment:
We don't need to explicitly close the file now that we're using a `File.open` block. The file is closed when the block ends.